DataSource/DataSourceBuilderInterface.php

```diff
@@ -6,18 +6,64 @@
 
 interface DataSourceBuilderInterface
 {
+    /**
```

Comment:
All these methods where docblock-ed to get rid of _"Method 'addField' not found in..."_ IDE warning in DataSourceConfiguration classes.